source/_integrations/lastfm.markdown

```diff
@@ -9,38 +9,13 @@ ha_domain: lastfm
 ha_platforms:
   - sensor
 ha_integration_type: integration
+ha_config_flow: true
 ---
 
 The `lastfm` sensor platform will allow you to see whenever a user starts scrobbling, their play count, last song played, and top song played on [Last.fm](https://www.last.fm/).
 
+{% include integrations/config_flow.md %}
+
 ## Setup
 
 To get an API key you need to create an [API account](https://www.last.fm/api/account/create).
```

Comment:
I think perhaps this should be above the config flow instructions.

Reply:
I just mean this sentence should go above the config flow section in the docs so users see this requirement before they click further to set it up. See [example](https://www.home-assistant.io/integrations/sensibo/#prerequisites)